docs/Troubleshooting/resolving-splitbrain.md

```diff
@@ -308,15 +308,312 @@ Healed gfid:b23dd8de-af03-4006-a803-96d8bc0df004.
 Number of healed entries: 3
 ```
 
+# 3.1 Resolution of GFID split-brain using gluster CLI
```

Comment:
Should we number this as 3.2 and add a 3.1 heading for the previous example for data/metadata split-brain resolution?

Reply:
Numbered the main heading as 3, and data/metadata heal as 3.1 and this as 3.2.